resources/js/modals/City/CreateCityModal.tsx

```diff
@@ -0,0 +1,46 @@
+import { createCity } from "~/api/cities";
+import type { ModalProps } from "~/shared.types";
+import { Button, Modal } from "~/ui";
+
+export const CreateCityModal = ({ show, onClose }: ModalProps) => {
+  async function handleSubmit(e: any) {
+    e.preventDefault();
+    try {
+      let res = await createCity({ name: e.target.name.value });
```

Comment:
```suggestion
      const res = await createCity({ name: e.target.name.value });
```

It's a good practice to only use let if you are going to change the variable value